types/office-js-preview/index.d.ts

```diff
@@ -13215,6 +13278,40 @@ declare namespace Office {
          * @param itemId - The Exchange Web Services (EWS) identifier for an existing calendar appointment.
          */
         displayAppointmentForm(itemId: string): void;
+        /**
+         * Displays an existing calendar appointment.
+         *
+         * The `displayAppointmentFormAsync` method opens an existing calendar appointment in a new window on the desktop or in a dialog box on
+         * mobile devices.
+         *
+         * In Outlook on Mac, you can use this method to display a single appointment that is not part of a recurring series, or the
+         * master appointment of a recurring series, but you cannot display an instance of the series.
+         * This is because in Outlook on Mac, you cannot access the properties (including the item ID) of instances of a recurring series.
+         *
+         * In Outlook on the web, this method opens the specified form only if the body of the form is less than or equal to 32KB number of characters.
```

Comment:
I'm a little confused here because the limit is in both KB and character count. If it's character count, consider the following for clarity.
```suggestion
         * In Outlook on the web, this method opens the specified form only if the body of the form has 32K characters or fewer.
```